src/utils/time.ts

```
@@ -48,6 +48,11 @@ export function getBroadcastTime(time: number): string {
 }
 
 export function getSessionTimestamp(time: number): string {
+  if (time === 0) {
+    // session without any messages
+    return ''
```

i don't really like silently returning empty string. we do this in a few places.

i think for utility functions that return a value we should either return "Type | null" or "Type" (but throw an error)